staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -36,9 +39,17 @@ import (
 func createLockObject(objectType string, objectMeta metav1.ObjectMeta) (obj runtime.Object) {
```

Comment:
Maybe change this to pass in the LeaderElectionRecord, rather than ObjectMeta, then create the record appropriately per lock resource?

Reply:
done